genai-perf/genai_perf/parser.py

```diff
@@ -286,6 +301,22 @@ def _set_artifact_paths(args: argparse.Namespace) -> argparse.Namespace:
     return args
 
 
+def parse_goodput(values):
+    constraints = {}
+    try:
+        for item in values:
+            target_metric, target_val = item.split(":")
+            constraints[target_metric] = float(target_val)
+    except ValueError:
+        raise argparse.ArgumentTypeError(
+            f"Invalid format for goodput constraints: {values}. "
```

Comment:
``
```suggestion
            f"Invalid format found for goodput constraints. "
```

Reply:
Updated all the text change. Skip replying to the rest for simplicity.